dist/.eslintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-	"extends": "../src/.eslintrc",
+	"extends": "../src/.eslintrc.json",
 	"rules": {
 		// That is okay for built version
```

Comment:
Yup

On Mon, Aug 1, 2016 at 6:20 PM, Oleg Gaidarenko <notifications@github.com>
wrote:

> In dist/.eslintrc.json
> <https://github.com/jquery/jquery/pull/3247#discussion_r73007213>:
>
> > @@ -1,5 +1,5 @@
> >  {
> > -	"extends": "../src/.eslintrc",
> > +	"extends": "../src/.eslintrc.json",
> >  	"rules": {
> >  		// That is okay for built version
>
> You mean add "the"?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jquery/jquery/pull/3247/files/ae11fce83a614e292174b80156ab3471fc944c09#r73007213>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABrUnoJ20lBcOagd1GDfDFgV_1RdGRzMks5qbhyzgaJpZM4JUqTq>
> .
>



-- 
Michał Gołębiowski